<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshFromRaster</class>
 <widget class="QDialog" name="MeshFromRaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create mesh from raster...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="interpretationGroupBox">
     <property name="title">
      <string>Interpret intensities as</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="4" column="0" colspan="2">
       <widget class="QRadioButton" name="ignoreButton">
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="otherButton">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="arrayNameEdit"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="elevationButton">
        <property name="text">
         <string>Elevation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="arrayNameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="materialButton">
        <property name="text">
         <string>Materials</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mshNameLabel">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>    Name of mesh:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mshNameEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="elementTypeGroupBox">
     <property name="title">
      <string>Represent pixels as</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="triButton">
        <property name="text">
         <string>sets of two triangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="quadButton">
        <property name="text">
         <string>rectangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="prismButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>set of two prisms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hexButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>hexahedra</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>elevationButton</tabstop>
  <tabstop>materialButton</tabstop>
  <tabstop>otherButton</tabstop>
  <tabstop>arrayNameEdit</tabstop>
  <tabstop>ignoreButton</tabstop>
  <tabstop>triButton</tabstop>
  <tabstop>quadButton</tabstop>
  <tabstop>hexButton</tabstop>
  <tabstop>mshNameEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeshFromRaster</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeshFromRaster</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
